tests/test_theme_alberta.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+import altair as alt 
+
+# Import theme_alberta
+from hueniversitypy.theme_alberta import *
+
+# Create toy dataframe to plot 
+data = pd.melt(pd.DataFrame({"A": [1, 86, 62, 29, 16, 42, 69],
+                             "B": [21, 41, 97, 14, 75, 7, 32],
+                             "C": [57, 90, 98, 67, 9, 46, 87]}))
+
+# Call the theme to apply themes to all subsequent plots
+alt.themes("theme_alberta", theme_alberta)
+alt.themes.enable("theme_alberta")
+
+# Create plot for testing
+alberta_plot = alt.Chart(data).mark_point().encode(
+    x="count()",
+    y="value",
+    color="variable"
+).properties(title="Test")
```

Comment:
I would remove this toy data from here and add it to all the test functions inside, so that the test work well..